src/pyrokinetics/gk_code/gene.py

```diff
@@ -178,27 +191,51 @@ def get_local_geometry(self) -> LocalGeometry:
                 self.data["geometry"].get("zeta", 0.0) != 0.0
                 or self.data["geometry"].get("zeta", 0.0) != 0.0
             ):
-                return self.get_local_geometry_miller_turnbull()
+                local_geometry = self.get_local_geometry_miller_turnbull()
             else:
-                return self.get_local_geometry_miller()
+                local_geometry = self.get_local_geometry_miller()
         elif geometry_type == "circular":
-            return self.get_local_geometry_circular()
+            local_geometry = self.get_local_geometry_circular()
         else:
             raise NotImplementedError(
                 f"LocalGeometry type {geometry_type} not implemented for GENE"
             )
 
+        # Need to get convention after?
+        if hasattr(self, "convention"):
+            convention = self.convention
+        else:
+            norms = Normalisation("get_local_species")
+            convention = getattr(norms, self.norm_convention)
+
+        local_geometry.normalise(norms=convention)
+
+        return local_geometry
+
     def get_local_geometry_miller(self) -> LocalGeometryMiller:
         """
         Load Miller object from GENE file
         """
+
         miller_data = default_miller_inputs()
 
         for (pyro_key, (gene_param, gene_key)), gene_default in zip(
             self.pyro_gene_miller.items(), self.pyro_gene_miller_default.values()
         ):
             miller_data[pyro_key] = self.data[gene_param].get(gene_key, gene_default)
 
+        minor_r = self.data["geometry"].get("minor_r", 0.0)
+        major_R = self.data["geometry"].get("major_r", 1.0)
+
+        if minor_r == 1.0:
+            self.norm_convention = "pyrokinetics"
+        elif major_R == 1.0:
+            self.norm_convention = "gene"
+        else:
+            raise ValueError(
+                "Pyrokinetics can only handle GENE simulations with either minor_r=1.0 or major_R = 1.0"
```

Comment:
I really like error messages that give you as much info as possible:
```suggestion
                f"Pyrokinetics can only handle GENE simulations with either minor_r=1.0 (got {minor_r}) or major_R = 1.0 (got {major_R})"
```